features/FxA-99-improved-login-notification/README.md

```diff
@@ -195,3 +195,21 @@ for inclusion when constructing the localized email.
   For example can users report a location that
   seems wildly inaccurate?
 
+### Results
```

Comment:
@rfk Wasn't sure if this should go in the `Outcomes` section. Currently, that section talks about high level tasks and requirements, not really the outcomes?